utils-cdi/src/test/java/be/atbash/util/FakeCDI.java

```diff
@@ -55,9 +60,13 @@ public BeanManager getBeanManager() {
 
     @Override
     public Instance<Object> select(Annotation... qualifiers) {
-        // TODO
-        throw new UnsupportedOperationException("Not implemented be.atbash.ee.jsf.jerry.util.cdi.FakeCDI.select(java.lang.annotation.Annotation...)");
 
+        if(qualifiers != null && qualifiers.length > 0)
+        {
```

Comment:
I always put { at the end of the line :)

Reply:
Right, code convention adopted